content/docs/getting-started/deploy-app/deploy-cli.md

```diff
@@ -14,4 +14,147 @@ seo:
   description: "" # custom description (recommended)
   canonical: "" # custom canonical URL (optional)
```

Comment:
the flag `toc: true` can be enabled to show the table of content on the right and side. This way a user can skip sections. What do you think?

Reply:
My bad - was not aware what the flag was doing